ocaml-lsp-server/src/compl.ml

```diff
@@ -128,5 +121,56 @@ let complete doc position ~markdown =
       let range = range_prefix lsp_position prefix in
       List.map ~f:(fun entry -> `Replace (range, entry)) entries
   in
-  let items = List.mapi ~f:(item ~markdown) items in
+  let textDocument =
+    TextDocumentIdentifier.create ~uri:(Document.uri doc |> Lsp.Uri.to_string)
+  in
+  let compl_info =
+    CompletionParams.(
+      create ~textDocument ~position:lsp_position ()
+      |> yojson_of_t |> Json.t_of_yojson)
```

Comment:
Is `Json.t_of_yojson` really necessary? It's just the identity function and it exists to just play nice with the code generators.

Reply:
You're right. Removed.